TodoList/Models/Item.swift

```diff
@@ -7,3 +19,20 @@ struct Item {
         self.isComplete = isComplete
     }
 }
+//NOTE: UserDefaults is a temporary solution
+extension Todos {
+    func save() {
+        if let encoded = try? JSONEncoder().encode(self) {
+            UserDefaults.standard.set(encoded, forKey: "todoList")
```

Comment:
Do we need to be careful about thread safety with UserDefaults?

Reply:
Good question! I don't know! lol. I will check..

Reply:
It looks like it is :) 

https://developer.apple.com/documentation/foundation/userdefaults
> Thread Safety
> The UserDefaults class is thread-safe.